### PR TITLE
[#166326149] add cosmos container messages

### PR DESCRIPTION
**Pull request by l-pinna**

### CODE REVIEW

File: westeurope/cosmosdb_container_messages/terraform.tfvars [hunk 1]
```diff
@@ -0,0 +1,36 @@
+terragrunt = {
+  dependencies {
+    paths = [
+      "../resource_group",
```
Comment (LucaPrete) [inline westeurope/cosmosdb_container_messages/terraform.tfvars]: remove the first two deps, as they are already deps of the third one. Also, remove comma from the end of the list

File: westeurope/cosmosdb_container_messages/terraform.tfvars [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+}
+
+# CosmosDB account specific variables
+cosmosdb_account_name = "apim"
+
+container_name = "messages"
+
+partitionKey_paths = ["/fiscalCode"]
+
+# azurerm_cosmosdb_collections = {
```
Comment (LucaPrete) [inline westeurope/cosmosdb_container_messages/terraform.tfvars]: remove commented/unused code

Comment (l-pinna) [reply westeurope/cosmosdb_container_messages/terraform.tfvars]: done

File: westeurope/cosmosdb_container_messages/terraform.tfvars (hunk 1 above)
Comment (l-pinna) [reply westeurope/cosmosdb_container_messages/terraform.tfvars]: done

**Pull request merged**